monitor-data/monitors.mdx

```diff
@@ -58,6 +59,33 @@ To create a threshold monitor, follow these steps:
 
 You have created a threshold monitor, and Axiom alerts you when the results from your query cross the threshold.
 
+## Anomaly monitors
+
+Anomaly monitors allow you to aggregate your event data and compare the results of this aggregation to what can be considered normal for the query. When the results are too much above or below the value that Axiom expects based on the event history, the monitor enters the alert state. The monitor remains in the alert state until the results no longer deviate from the expected value. An anomaly monitor sends you a notification each time it enters or exits the alert state.
```

Comment:
```suggestion
Anomaly monitors allow you to aggregate your event data and compare the results of this aggregation to what can be considered normal for the query. When the results are too much above or below the value that Axiom expects based on the event history, the monitor enters the alert state. The monitor remains in the alert state until the results no longer deviate from the expected value. Note that this can happen without the results returning to their previous level, if they stabilise around a new value. An anomaly monitor sends you a notification each time it enters or exits the alert state.
```